.github/workflows/cross.yml

```diff
@@ -21,62 +21,67 @@ env:
 jobs:
   aws-lc-rs-cross-test:
     if: github.repository_owner == 'aws'
-    name: cross tests ${{ matrix.target }}
+    name: cross tests ${{ matrix.target[0] }}
     runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
         target:
-          - aarch64-linux-android
-          - aarch64-unknown-linux-gnu
-          - aarch64-unknown-linux-musl
-          - arm-linux-androideabi
-          - arm-unknown-linux-gnueabihf
-          - armv7-linux-androideabi
-          - armv7-unknown-linux-gnueabihf
-          - i686-unknown-linux-gnu
-          - powerpc-unknown-linux-gnu
-          - powerpc64-unknown-linux-gnu
-          - powerpc64le-unknown-linux-gnu
-          - riscv64gc-unknown-linux-gnu
-          - s390x-unknown-linux-gnu
-          - x86_64-pc-windows-gnu
-          - x86_64-unknown-linux-musl
+          - [aarch64-linux-android, 0]
+          - [aarch64-unknown-linux-gnu, 1]
+          - [aarch64-unknown-linux-musl, 1]
+          - [arm-linux-androideabi, 0]
+          - [arm-unknown-linux-gnueabihf, 0]
+          - [arm-unknown-linux-musleabi, 1]
+          - [arm-unknown-linux-musleabihf, 1]
+          - [armv7-linux-androideabi, 0]
+          - [armv7-unknown-linux-gnueabihf, 0]
+          - [i686-unknown-linux-gnu, 0]
+          - [powerpc-unknown-linux-gnu, 1]
+          - [powerpc64-unknown-linux-gnu, 1]
```

Comment:
what's the reasoning for which are tested w/ FIPS and which aren't?

Reply:
I'm only enabling the FIPS testing for platforms for which customers have requested.